package.json

```diff
@@ -1,63 +1,75 @@
 {
-  "name": "web3-react-boilerplate-private",
+  "name": "web3-react-boilerplate",
   "private": true,
   "version": "0.0.0",
   "type": "module",
   "license": "MIT",
   "author": "Wonderland",
+  "engines": {
+    "node": ">=18.17.0"
+  },
   "scripts": {
-    "dev": "vite",
-    "build": "tsc && vite build",
-    "preview": "vite preview",
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "preview": "next start",
     "lint": "eslint 'src/**/*.{js,jsx,ts,tsx}'",
     "lint:fix": "yarn run lint -- --fix",
     "prettier": "prettier src --check",
     "prettier:fix": "yarn run prettier -- --write",
     "format": "yarn run prettier:fix && yarn run lint:fix",
     "format:check": "yarn run prettier && yarn run lint",
     "prepare": "husky install && wonderland-crypto-husky-checks install",
-    "test": "vitest run --dir src/tests/",
-    "test:watch": "vitest --dir src/tests/"
+    "test": "yarn cypress run",
+    "test:watch": "yarn cypress open"
   },
   "lint-staged": {
     "src/**/*.{js,jsx,ts,tsx}": "eslint --cache --fix",
     "src/**/*.{js,jsx,ts,tsx,css,scss,md,html}": "prettier --write --ignore-unknown"
   },
   "dependencies": {
-    "@rainbow-me/rainbowkit": "1.0.9",
+    "@emotion/cache": "11.11.0",
+    "@emotion/react": "11.11.3",
+    "@emotion/server": "11.11.0",
+    "@emotion/styled": "11.11.0",
+    "@mui/icons-material": "5.15.6",
+    "@mui/material": "5.15.14",
+    "@mui/material-nextjs": "5.15.11",
+    "@next/eslint-plugin-next": "14.1.3",
+    "@rainbow-me/rainbowkit": "2.0.2",
+    "@tanstack/react-query": "5.28.0",
+    "next": "14.1.3",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-router-dom": "6.15.0",
-    "viem": "1.19.15",
-    "wagmi": "1.4.12"
+    "viem": "2.8.6",
+    "wagmi": "2.5.7"
   },
   "devDependencies": {
     "@defi-wonderland/crypto-husky-checks": "1.3.0",
     "@testing-library/jest-dom": "6.1.2",
-    "@testing-library/react": "14.0.0",
+    "@testing-library/react": "14.2.1",
     "@testing-library/user-event": "14.4.3",
     "@types/lodash.merge": "4.6.7",
     "@types/react": "18.2.15",
     "@types/react-dom": "18.2.7",
     "@typescript-eslint/eslint-plugin": "6.0.0",
     "@typescript-eslint/parser": "6.0.0",
-    "@vitejs/plugin-react": "4.0.3",
+    "cypress": "13.7.0",
     "eslint": "8.45.0",
     "eslint-config-prettier": "9.0.0",
+    "eslint-formatter-codeframe": "7.32.1",
```

Comment:
is this package not deprecated?
Also, idk what this package does very well. Since we are using prettier to format, the name of this package suggest that it formats code too, but we don't need any more format right?

Reply:
removed, not needed https://github.com/defi-wonderland/web3-react-boilerplate/pull/8/commits/a4a80bcb3d05cb213b045a2f40932f8ae160615c